tests/test_ckan_to_frictionless.py

```diff
@@ -264,6 +264,56 @@ def test_dataset_extras(self):
             'location': {'country': 'China'},
         }
 
+    def test_unjsonify_all_extra_values_in_nested_dicts(self):
+        self.dataset_dict.update({
```

Comment:
let's not self.dataset stuff - let's just create an explicit dict

Reply:
Ok, that makes sense. Would you like me to make it explicit everywhere (and thus remove completely the assignment of `self.dataset_dict` and `self.resource_dict` at the top of the class in `setup_class`) or should we keep what we have as it is?